### PR TITLE
[doc] Update the document about offline cache

### PR DESCRIPTION
**Pull request by PGZXB**
Issue: #6263, #4401

### CODE REVIEW

File: docs/lang/articles/performance_tuning/performance.md [hunk 1]
```diff
@@ -241,7 +241,17 @@ We address this problem by introducing the *offline* cache feature, which dumps
 To verify the effect, run some examples twice and observe the launch overhead:
 ![](../static/assets/effect_of_offline_cache.png)
 
-For now, the offline cache feature works *only* on the CPU and CUDA backends.
+Supported backends:
+| Backend | Supported or not |
+| ----------- | ----------- |
+| CPU     | ✔ |
```
Comment (strongoier) [inline docs/lang/articles/performance_tuning/performance.md:247]:
```suggestion
| CPU     | :heavy_check_mark: |
```

File: docs/lang/articles/performance_tuning/performance.md [hunk 2]
```diff
@@ -241,7 +241,17 @@ We address this problem by introducing the *offline* cache feature, which dumps
...
+| CUDA |  ✔  |
+| Vulkan |  ✔  |
+| OpenGL |  ✔  |
+| Metal |  ✔ |
+| CC  |  ❌ |
```
Comment (strongoier) [inline docs/lang/articles/performance_tuning/performance.md:252]:
```suggestion
| CC  | :x: |
```

File: docs/lang/articles/performance_tuning/performance.md [hunk 3]
```diff
@@ -241,7 +241,17 @@ We address this problem by introducing the *offline* cache feature, which dumps
 To verify the effect, run some examples twice and observe the launch overhead:
 ![](../static/assets/effect_of_offline_cache.png)
 
-For now, the offline cache feature works *only* on the CPU and CUDA backends.
+Supported backends:
+| Backend | Supported or not |
```
Comment (strongoier) [inline docs/lang/articles/performance_tuning/performance.md:245]:
```suggestion

| Backend | Supported or not |
```

Comment (strongoier) [reply docs/lang/articles/performance_tuning/performance.md:245]: A blank line is needed to successfully show the table.

**Comment by strongoier** [comment]
Do we need to mention CC & DX backends? It seems that https://docs.taichi-lang.org/docs/master/hello_world#supported-systems-and-backends doesn't mention them. cc: @ailzhang @neozhaoliang @Vissidarte-Herman

**Comment by ailzhang** [comment]
nope it's totally fine to skip them for now :D

**Comment by strongoier** [comment]
Then @PGZXB maybe we can simply remove the table?

**Comment by PGZXB** [comment]
> Then @PGZXB maybe we can simply remove the table?

And remove "For now, the offline cache feature works *only* on the CPU and CUDA backends." too?

**Comment by strongoier** [comment]
> And remove "For now, the offline cache feature works _only_ on the ..." too?

Yep.

**Review by strongoier — APPROVED** [review]
Thanks!

**Pull request merged**